chapters/ch10.md

```diff
@@ -1,23 +1,25 @@
 # Having a dev set and metric speeds up iterations
 ->
-
+# Xây dựng một tập phát triển và một phép đo sẽ thúc đẩy nhanh quá trình làm việc.
 It is very difficult to know in advance what approach will work best for a new problem. Even experienced machine learning researchers will usually try out many dozens of ideas before they discover something satisfactory. When building a machine learning system, I will often:
 ->
-
+Thật sự rất khó để biết trước hướng đi nào sẽ phù hợp cho một vấn đề mới. Kể cả những nghiên cứu sinh Machine Learning nhiều kinh nghiệm cũng thường thử nghiệm một vài ý tưởng trước khi có thể khám phá ra một cái gì đó thỏa đáng. Khi xây dựng một hệ thống Machine Learnig, tôi sẽ thường: 
 1. Start off with some ​**idea​** on how to build the system.
 ->
-
+Bắt đầu với một số ý tưởng nào đó về cách xây dựng hệ thống đó.
 2. Implement the idea in ​**code​**.
 ->
-
+Hiện thực hóa ý tưởng đó dưới dạng code.
```

Comment:
> Hiện thực hóa ý tưởng ~~đó~~ dưới dạng code.

Reply:
ở đây có "the" thì là đã trỏ đến một idea được nhắc đến ở bên trên, nên để chữ "đó" đi trước ý tưởng cũng ko sai ạ.